core-plugin/src/com/google/gct/idea/debugger/ui/CloudAttachDialog.java

```diff
@@ -156,6 +159,7 @@ protected JComponent createCenterPanel() {
   @Override
   protected void doOKAction() {
     if (getOKAction().isEnabled()) {
+      doTracking();
```

Comment:
shouldn't all the tracking cases in doTracking be inlined into the isSelected block and syncOrStash?